contracts/evmscript/EVMScriptRunner.sol

```diff
@@ -16,13 +16,13 @@ contract EVMScriptRunner is AppStorage, EVMScriptRegistryConstants {
 
     function runScript(bytes _script, bytes _input, address[] _blacklist) protectState internal returns (bytes output) {
         // TODO: Too much data flying around, maybe extracting spec id here is cheaper
-        address executorAddr = getExecutor(_script);
-        require(executorAddr != address(0));
+        IEVMScriptExecutor executorAddr = getExecutor(_script);
+        require(address(executorAddr) != address(0));
```

Comment:
Do we need to explicitly cast to `address` here?

Reply:
For now, no, but it actually looks like in [`pragma 0.5.0`, the implicit `address` to `Contract` inheritance will change (and so we will need explicit casting)](https://solidity.readthedocs.io/en/v0.4.24/types.html#address).

This probably also means the type checker won't let us get away with the implicit conversion in function arguments, but let's save that for when we do move up.